Project.toml

```diff
@@ -9,11 +9,13 @@ EllipsisNotation = "da5c29d0-fa7d-589e-88eb-ea29b0a81949"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 FieldMetadata = "bf96fef3-21d2-5d20-8afa-0e7d4c32a885"
 Future = "9fa8497b-333b-5362-9e8d-4d0656e87820"
+GaussianRandomFields = "e4b2fa32-6e09-5554-b718-106ed5adafe9"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
```

Comment:
This isn't used here, is it?

Reply:
Yeah sorry I just noticed this was incorrectly added; now removed. I think I added this when I initially got a missing dependency error when running the tests before realising I was using the wrong environment.